format/spec.md

```diff
@@ -1370,3 +1370,16 @@ Writing v2 metadata:
     * `sort_columns` was removed
 
 Note that these requirements apply when writing data to a v2 table. Tables that are upgraded from v1 may contain metadata that does not follow these requirements. Implementations should remain backward-compatible with v1 metadata requirements.
+
+## Appendix F: Implementation Notes
+
+This section covers topics not required by the specification but recommendations for systems implementing the Iceberg specification
+to help maintain a uniform experience.
+
+### Point in Time Reads (Time Travel)
+
+Iceberg supports two types of histories for tables. A history of previous "current snapshots" stored in ["snapshot-log" table metadata](#table-metadata-fields) and [parent-child lineage stored in "snapshots"](#table-metadata-fields). These two histories 
```

Comment:
What is exactly meant by `two histories might not be consistent for determining a table state`?

Reply:
tried to clarify.  please let me know if this reads better.

Reply:
Gentle ping @aokolnychyi

Reply:
> The discrepancies can be caused by a variety of table operations (e.g. updating the `current-snapshot-id` of the table).

might be better to further clarify the discrepancy. this wasn't super clear to me.

Reply:
I added more lanugage here, please let me know if it addresses the concerns.